packages/terra-action-header/src/ActionHeader.jsx

```diff
@@ -196,7 +234,7 @@ const ActionHeader = ({
   return (
     <ActionHeaderContainer
       {...customProps}
-      startContent={leftButtons}
+      startContent={leftButtons || visuallyHiddenComponent}
```

Comment:
Is this what is causing the title in the action header to shift to the left in the screenshots. It looks like there isn't much left padding in the new screenshots. Is this expected?

Reply:
Yes the visually-hidden-text has a width of 1px which is causing a slight shift.

Reply:
Is this approved by UX? Otherwise, I think we need to adjust the styles to make it look how it used to be.

One possible option is to add the appropriate padding/margin to this instance of `visuallyHiddenComponent` to match how it used to be.

Reply:
Putting the visually hidden content in the startContent is the wrong location to insert. By including it here is it what is causing the current screenshot changes and the look is undesired UX changes. 

We intentionally created different left margin alignments depending on if there are action-header buttons or not in the start content, shifting between 10px if no contents, to 5px if buttons are present (has to do with how the no border buttons visually look to best align). The css that applies these differences is here: https://github.com/cerner/terra-core/blob/a7c88211833e8f180dec97341ef28bbdc590355e/packages/terra-action-header/src/ActionHeaderContainer.module.scss#L69

I would recommend inserting the visually hidden components within their own 0 height parent row, that will render on its own row below the entire current action-header contents flex row.